File: DSA/Cpp/linearSearch.cpp
#include <iostream>
using namespace std;

int main(){
    int arr[5] = {19, 14, -5, 17, 21};
    int target;

    cin>>target;

    for (int i = 0; i < 5; i++){
        int element = arr[i];
        if (element == target){
            cout<<"Element is found at index: "<<i + 1;
        }
    }
    return -1;
}

File: DSA/Python/Linear Search.py
def linear_search(arr, key):
    for i in range(len(arr)):
        if arr[i] == key:
            return i  # Returns the index of the key element
    return -1  # Return -1 if key is not found


arr = [10, 23, 45, 70, 11, 15]
key = 70
result = linear_search(arr, key)

if result != -1:
    print(f"Element found at index {result}")
else:
    print("Element not found")